server/api.go

```diff
@@ -95,13 +104,33 @@ func (p *Plugin) connectUserToGitHub(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
-	conf := p.getOAuthConfig()
+	privateAllowed := false
+	pValBool, err := strconv.ParseBool(r.URL.Query().Get("private"))
```

Comment:
Any objections on logging `err` if it's `!= nil`?

Reply:
No objection, per se, but why? That would mean the user intentionally edited the connect link to put in something non-true or non-false. That's not really an error, and it should default to false in that case. 
Sound good?

Reply:
I'd go one step furhter and just write this as:
```
	pValBool, _ := strconv.ParseBool(r.URL.Query().Get("private"))
```

Reply:
While I have to agree in this specific case that ignoring the error is fine, but in general it's a non idiomatic pattern. In the future there might be other sources that hit this endpoint besides the connect link and then it would be handy to know if an error happens.

Reply:
To have a bit more background: I've run into a couple of cases in the past with this plugin and others where I tried debugging a failure, but the error was not logged. Because of this I'm general advocating for logging as many errors as possible.